spec/unit/action/runner_spec.rb

```diff
@@ -230,14 +230,6 @@ def call
         runner.setup_settings
         runner.setup_authorization
       end
-
-      it 'errors if no custom forge URL is set' do
```

Comment:
Would it be valuable to test that setting `authorization_token` now _does_ work if `baseurl` isn't set?

Reply:
Considering how mocked up this all is, not really IMO.